app/screens/route-list/route-list-screen.tsx

```diff
@@ -76,7 +86,21 @@ export const RouteListScreen = observer(function RouteListScreen({ navigation, r
     return fontScale <= 1.2 && deviceWidth >= 360 && shouldShowDashedLineByTextLength
   }, [trains.data])
 
-  const renderRouteCard = ({ item }: { item: RouteItem }) => {
+  const renderRouteCard = ({ item }: { item: RouteData }) => {
+    if (typeof item === "string") {
+      return (
+        <View
+          style={{
+            display: "flex",
+            alignItems: "center",
+            padding: "2%",
+            justifyContent: "center",
+          }}
```

Comment:
Move to styles object at the top of the file. Also, don't use `%` in padding

Reply:
🤙🏻